### PR TITLE
Creates layout and fragment for MPU925X

### PR DESCRIPTION
**Pull request by asitava1998**
Fixes issue #409 

Changes: 
* Creates layout and fragment for MPU925X.

Issue:
* MPU6050 and MPU925X share the same address, distinguishing them at the device level is not possible. (It's possible for Arduino, but I don't know about PSLab).
* if it is not possible, we have to rely solely on the user to select the right sensor. This can be taken up as a separate issue.

Screenshots for the change:

### CODE REVIEW

**Comment by akarshan96** [comment]
@asitava1998 can you see what is causing the build to fail. Thanks.

**Comment by asitava1998** [comment]
@akarshan96 I updated the branch and then the build failed. I am looking into it.

**Comment by asitava1998** [comment]
@akarshan96 @viveksb007 @CloudyPadmal Can we speed up the review process? The files in this PR are also present in other PRs and when those are merged, it leads to merge conflicts here.

File: app/src/main/java/org/fossasia/pslab/sensorfragment/SensorFragmentMPU925X.java [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+
+            dataset1.setColor(Color.BLUE);
+            dataSet2.setColor(Color.GREEN);
+            dataSet3.setColor(Color.RED);
+
+            dataset4.setColor(Color.BLUE);
+            dataSet5.setColor(Color.GREEN);
+            dataSet6.setColor(Color.RED);
+
+            dataset1.setDrawCircles(true);
```
Comment (viveksb007) [inline app/src/main/java/org/fossasia/pslab/sensorfragment/SensorFragmentMPU925X.java]: ```drawCircles``` are ```true``` by default, if you want to not draw circle at data point then you set it to ```false```.

**Comment by CloudyPadmal** [comment]
Looks OK! :) Once the requested changes by Vivek is made I'll approve this! :+1:

File: app/src/main/java/org/fossasia/pslab/sensorfragment/SensorFragmentMPU925X.java (hunk 1 above)
Comment (asitava1998) [inline app/src/main/java/org/fossasia/pslab/sensorfragment/SensorFragmentMPU925X.java]: Changes done

Comment (viveksb007) [inline app/src/main/java/org/fossasia/pslab/sensorfragment/SensorFragmentMPU925X.java]: What have you changed?
```dataset.setDrawCircles(true)``` doesn't make any difference, it true by default.

**Pull request merged**